index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    Hello World, we are software developers!
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
- [ ] [required] I think it would be nice if you remove this commented codes, in that way your code will be much more clean and readable

Reply:
> 
> 
> Hi @descholar-ceo,
> 
> Good job so far!
> There are some issues that you still need to work on to go to the next project but you are almost there!
> ## Required Changes ♻️
> 
> _Check the comments under the review._
> 
>     * [ ]  You can use [this template](https://github.com/microverseinc/readme-template) for readme file
> 
>     * [ ]  Make sure that you fix all the points in inline comments
> 
> 
> ## Optional suggestions
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> You can also consider:
> 
>     * [ ]  It would be very nice if you add a live demo link in readme file
> 
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @descholar-ceo,

## To Highlight
Thank you for conducting this review, I really liked that you have used inline comments to tell the student specifically where they need to change or add the code rather than in general.

## To improve
Let's make sure to cover the following aspects for future reviews:

- **Check that the repository looks professional**: The student has not checked the linters you should've invalidated it and ask them to set up the linter file for this project with this [given link](https://github.com/microverseinc/linters-config).
- Also you should have advised to not commit files that are not related to the project. Instead, add them to a .gitignore file. (ex: .vscode)

That's it!
Thank you for performing this review and I hope mine made sense to you.
Happy Reviewing!

**_Dipesh Kumar_**